chart/README.md

```diff
@@ -15,3 +15,15 @@ helm init
 helm install --name kubeless --namespace kubeless ./kubeless
 ```
 
+After that,Please go to docs/misc,Select an appropriate pv for kafka component.
```

Comment:
suggestion for the text:

---

Depending on the cluster configuration it is possible that persistent volumes for Kafka and Zookeeper are not provisioned by default. If that is your case both stateful sets will remain as "Pending":

```
$ kubectl get pods -n kubeless
NAME                                           READY     STATUS    RESTARTS   AGE
kafka-0                                        1/1       Pending   0          1h
kafka-trigger-controller-7f4f458f8b-l6f5m      1/1       Running   0          1h
kubeless-controller-manager-58d78fff74-g7fsd   1/1       Running   0          1h
zoo-0                                          1/1       Pending   0          1h
``` 

If that is your case refer to [this documentation](/docs/troubleshooting/#kafka-and-zookeeper-persistent-volume-creation) for provision those volumes manually.

Reply:
Then in the section troubleshooting section [here](/docs/troubleshooting/#kafka-and-zookeeper-persistent-volume-creation) you can add a small note saying that people can use those manifests:

---

If you are running Kubernetes in GKE check the specific guide [here](/docs/GKE-deployment) to create the required disks and PVs. **You can also provision those persistent volumes manually deploying the manifests present in the [misc folder](https://github.com/kubeless/kubeless/tree/master/docs/misc)**. In other case, check the provider documentation of how to create these required volumes. Note that `kafka` and `zookeeper` are only needed when working with Kafka events, you can still use Kubeless to trigger functions using HTTP requests.

Reply:
thanks for your suggestion

Reply:
i removed kafka-pv-hostpath.yaml.i notice user refer to kubernetes docs to create appropriate PV.